SimulationConditions/src/SimBookkeeper.cc

```diff
@@ -0,0 +1,23 @@
+//
+// SimBookkeeper is the ProditionsEntry for
```

Comment:
Another incomplete comment.   By the way,   if the purpose of a class is explained in the header, I don't think it needs to be repeated in the implementation file.

Reply:
Removed since I added it in the header (commit 354fb8ff2)